test/useObserver.test.tsx

```diff
@@ -3,7 +3,9 @@ import * as mobx from "mobx"
 import * as React from "react"
```

Comment:
great tests! Nit; it might nice to put them in a separate test file as they are addressing a very specific problem